3.-Web-api-call-Microsoft-graph-for-personal-accounts/README.md

```diff
@@ -267,194 +224,120 @@ Explore the sample by signing in into the TodoList client, adding items to the T
 
 NOTE: Remember, the To-Do list is stored in memory in this `TodoListService-v2` sample. Each time you run the TodoListService API, your To-Do list will get emptied.
 
-### Current limitations
-
-The on-behalf-of flow works for Microsoft Personal accounts, but the consent is not yet rolled-up in the client for the user to consent to the Web API calling the downstream API (here Microsoft Graph). To make this work, the suggestion is:
-
-- either to use the same client ID in the Client and the Service. This way the consent for the service will appear in the client.
-- or to provide a protected page on the Web API (which therefore also becomes a Web app) so that the user can have an interaction
-
 ## How was the code created
 
-For details about the way the code to protect the Web API was created, see [How was the code created](../1.%20Desktop%20app%20calls%20Web%20API/README.md#How-was-the-code-created) section, of the README.md file located in the sibling folder named **1. Desktop app calls Web API**.
-
-This section, here, is only about the additional code added to let the Web API call the Microsoft Graph
-
-### Reference MSAL.NET
+For details about the way the code to protect the Web API was created, see [How was the code created](../.%20Web%20API%20now%20calls%20Microsoft%20Graph#How-was-the-code-created) section, of the README.md file located in the sibling folder named **2. Web API now calls Microsoft Graph**.
 
-Calling a downstream API involves getting a token for this Web API. Acquiring a token is achieved by using MSAL.NET.
+This section, here, is only about the differences in the code to let the Web API call the Microsoft Graph with Microsoft personal accounts
```

Comment:
this is a lot of words here...suggestion: This section addresses the differences in the code for the Web API calling the Microsoft Graph with Microsoft personal accounts"